src/main/java/org/folio/repository/UserTenantRepository.java

```diff
@@ -28,7 +28,7 @@ public class UserTenantRepository {
   public static final String BARCODE = "barcode";
   public static final String EXTERNAL_SYSTEM_ID = "external_system_id";
   public static final String CONSORTIUM_ID = "consortium_id";
-  public static final String LOWERCASE_WRAPPED_USERNAME = String.format("LOWER(%s)", USERNAME_FIELD);
+  public static final String LOWERCASE_WRAPPED_USERNAME = String.format("LOWER(f_unaccent(%s))", USERNAME_FIELD);
```

Comment:
Modified this query to use indexes, for testing I created 1k records on rancher and run this query without f_unaccent
![image](https://github.com/folio-org/mod-users/assets/25097693/ce709704-62de-49fa-8a6f-a8c252a4cce8)
And the same query but with adding f_unaccent
![image](https://github.com/folio-org/mod-users/assets/25097693/234119b1-6d54-4232-91b1-f01811b5ffdd)